cob_moveit_bringup/package.xml

```diff
@@ -22,5 +22,7 @@
   <exec_depend>moveit_setup_assistant</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>tf</exec_depend>
+  <exec_depend>IndustrialFCL</exec_depend>
+  <exec_depend>stomp_moveit</exec_depend>
```

Comment:
also add `moveit_ros_perception` as `occupancy_map_monitor` as it is used in cob_robots/cob_moveit_config sensors.yaml!